ci/jenkins/README.md

```diff
@@ -85,27 +85,27 @@ DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
 ./ci/jenkins/test.sh --testcase networkpolicy --registry ${DOCKER_REGISTRY}
 ```
 
-* windows e2e: e2e tests in a Windows setup.
+* windows e2e: e2e tests in a Windows setup with Docker runtime.
 
 ```shell
 #!/bin/bash
 DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
 ./ci/jenkins/test.sh --testcase windows-e2e --registry ${DOCKER_REGISTRY}
 ```
 
-* windows conformance: community tests on Windows cluster, focusing on "Conformance|sig-windows" and "sig-network",
-  and skipping "LinuxOnly", "Slow", "Serial", "Disruptive", "Flaky", "Feature", "sig-cli", "sig-storage", "sig-auth",
-  "sig-api-machinery", "sig-apps", "sig-node", "Privileged", "should be able to change the type from", "[sig-network]
-  Services should be able to create a functioning NodePort service [Conformance]", "Service endpoints latency should not
-  be very high".
+* windows conformance: community tests on Windows cluster with Docker runtime, focusing on "Conformance|sig-windows" and
```

Comment:
not sure why we include so many "details" here; this seems hard to keep up-to-date